css/parser/src/lexer/mod.rs

```diff
@@ -430,6 +408,83 @@ where
         }
     }
 
+    // Consume comments.
+    // This section describes how to consume comments from a stream of code points.
+    // It returns nothing.
+    fn read_comments(&mut self) -> LexResult<()> {
+        // If the next two input code point are U+002F SOLIDUS (/) followed by a U+002A
+        // ASTERISK (*), consume them and all following code points up to and including
+        // the first U+002A ASTERISK (*) followed by a U+002F SOLIDUS (/), or up to an
+        // EOF code point. Return to the start of this step.
+        // NOTE: We allow to parse line comments under the option.
+        let is_block_comment = self.input.cur() == Some('/') && self.input.peek() == Some('*');
+        let is_inline_comment = self.config.allow_wrong_line_comments
```

Comment:
Is this variables required?
I think `rustc` may optimize calls to `self.input.cur()` if `is_block_comment` is true, but I think checking these at each if statement will be better.

Reply:
Agree, fixed